srcpkgs/fwupd/template

```diff
@@ -27,7 +22,7 @@ maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="LGPL-2.1-or-later"
 homepage="https://github.com/hughsie/fwupd"
 distfiles="https://github.com/hughsie/fwupd/archive/${version}.tar.gz"
-checksum=4abecf930f02fc5b165831682f3915cc7f0e82e73be2e187e761220ae275a1a3
+checksum=f889e96c006f55df185e07faf5257b4ae93a5a8ed5cecb1afbab79807f8918e5
 replaces="fwupdate>=0"
 
 case "$XBPS_TARGET_MACHINE" in
```

Comment:
Below this, there are some options for dealing with `_have_uefi`. You should fix them so they stop bringing in the dependencies like `libefivar-devel`, and instead only add `fwupd-efi` to dependencies. But it shouldn't be added at the top for all archs.

Reply:
Let me keep everything for the moment, and then start eliminating stuff after we fix the gi-docgen error.

Reply:
Actually, they all seem to be required as in present state.

Reply:
Ok, thanks for checking!

Reply:
Actually, `efivar` and `gnu-efi-libs` could be removed, which I did, and it still builds fine.
Had to keep the new `fwupd-efi` in both deps and makedeps though, otherwise it would fail or not pick it up as deps.